dev/protos/google/firestore/proto/bundle.proto

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Google LLC.
```

Comment:
We need to find a more descriptive (or at least less repetitive) path for this file :) What about simply `dev/protos/firestore/bundle.proto`? If you want to keep this under `google` (which I am not all that excited about), then I would make it `dev/protos/google/firestore/local/bundle.proto`.

Reply:
Done.